assessments/0002-notary-project.md

```diff
@@ -0,0 +1,195 @@
+   - [Notary V2 prototype cli](https://github.com/notaryproject/nv2)
+- The [V1 Notary Overview](https://github.com/theupdateframework/notary#overview) is reasonable and explanatory.
+- The [Notary V2 Overview](https://github.com/notaryproject/notaryproject#notary-v2-overview) provides a good explanation of the V2 project, but which is aimed at the specification writers/developers.
+
+- Is the documentation feature complete? (i.e., each product feature is documented)
+   - As the project is in ongoing development, not every feature is completed and so there are some holes in the documentation.
+   - The https://github.com/notaryproject/nv2 repository contains the nv2 prototype cli as well as the documentation about how to use it. Again, since the project is in development it seems like there are some holes. 
+
+- Are there step-by-step instructions (tasks, tutorials) documented for features?
+   - Since V2's documentation is so conceptual at this point (consisting mostly of specification), it may be best to leave it as is -- linking to it directly during the development process, and planning documentation along side the development of the system as a whole.
+
+- Is the “happy path”/most common use case documented.
+   - The V1 getting started guide is very good, will V2 be able to draw from it?
+
+- If the documentation does not suffice, is there a clear escalation path for users needing more help? (FAQ, Troubleshooting)
+   - This is an area for improvement on both V1 and V2 docs. Potentially consider reorganizing the whole docset with users in mind: dividing the content in a concepts, tasks, and tutorials model. 
+
+#### New user content
+
+- Getting Started Guide is clearly labelled: https://github.com/theupdateframework/notary#getting-started-with-the-notary-cli though notably only in v1 of their docs, not v2. How does one get started with v2?
+- The v2 overview has getting started/CLI steps for an entirely different set of functions: https://github.com/notaryproject/notaryproject#notary-v2-overview. Are these in addition to or in replacement of the v1 getting started steps?
+- Where do users go after the getting started steps? Is there anything else about the tool they need to understand?
+- Sample code looks great!
+
+#### Content maintainability
+
+- As the documentation is in GitHub it is searchable. When setting up a website, we recommend using Google, Algolia, or Lunr to provide search functionality. 
+  
+- Is the Notary Project planning on localization/internationalization? If so, we recommend starting the site with a directory structure which supports localization (Hugo supports this very well out of the box).
+
+- Is any of the content from version 1 relevant to version 2? 
+  - If so, import v1 into the site, and "version" by copying the content into a new directory for v2 and updating only what needs to change or be added.
+  - If not, upload v1 and v2 into the site as separate document sets. 
```

Comment:
When referring to versions, a consistent way of writing "version", "v#", or "V#" helps reduce cognitive load, teaching the pattern with the first instance. (see line 82 vs lines 83/84)